data/scenario-02.json

```diff
@@ -66,8 +66,8 @@
                 "blockId": "f14d8799-d0dd-24d8-ca19-1cc5812b704e",
                 "type": "rawImage",
                 "model": {
-                  "width": 640,
-                  "height": 420,
+                  "width": 572,
```

Comment:
```
"width": 2362,
"height": 2952,
```

Reply:
This is a Visual Journalism image that has the extension -nc.jpg. That means that it's not compressed by image chef currently so as to retain its quality. i.e as sen in this story: https://www.bbc.co.uk/news/uk-43826163. Therefore I left its dimensions as we would receive them.